docs/experimental/examples/transferring_data/submit_transfer/index.rst

```diff
@@ -0,0 +1,54 @@
+
+Initiating a Transfer
+=====================
+
+Moving data within the Globus Ecosystem can be performed by submitting a `Transfer Task`
+to the Globus Transfer service.
+
+The below example demonstrates how to submit that task using a ``TransferClient``.
+
+.. note::
+    The below example is for moving data between two publicly hosted "tutorial"
+    collections.
+    You should replace these collection ids and path values with your own collection
+    ids and paths.
+
+.. note::
+    Some collections require you to attach a "data_access" scope to your transfer
+    request. You should evaluate whether this is necessary for both your source and
+    destination collections and omit the ``transfer_client.add_app_data_access_scope``
+    calls as needed.
+
+    A collection requires "data_access" if it is (1) a mapped collection and (2) is
+    not high assurance.
+
+    If you're still not able to determine, an alternative reactive script is included
+    below.
+
+.. literalinclude:: submit_transfer.py
+    :caption: ``submit_transfer.py`` [:download:`download <submit_transfer.py>`]
+    :language: python
+
+
+Reactively solving ConsentRequired errors
+------------------------------------------
+
+Some collections require you to attach a "data_access" scope to your transfer
+request.
+
+This can be typically be done proactively: if you know which collections you or your
```

Comment:
I would not characterize either case as typical. In my experience, our users break into three basic groups:
- those who are writing tools against fixed collections (both ends fixed)
- those who are writing tools against _one_ fixed collection (one end fixed, one end open)
- those who are writing generic tools (both ends open)

And the latter two groups are more similar than the first.

If anything, knowing both of your collection IDs ahead of time is the exception, not the rule.

Do I object to the ordering used in this doc, or the overall presentation? _Eh._ Definitely not strongly, but this note jumps out at me and makes me feel that something is slightly amiss. I'm not sure, but I think that we should actually have a preamble that says:
- if you know one or both of your collection IDs, do it _this way_
- if you don't know the collection IDs, do it _that way_

with intra-document links.

Reply:
Hmm, I can take another stab at this doc with the preamble you suggest.

Reply:
Updated the doc to be less assumptive; lemme know what you think now.